src/EventManagement.Web/Startup.cs

```diff
@@ -169,11 +169,18 @@ public virtual void ConfigureServices(IServiceCollection services)
                 services.Configure<PowerOfficeOptions>(Configuration.GetSection("PowerOffice"));
                 services.AddScoped<IInvoicingService, PowerOfficeService>();
             }
+            else if(appsettings.UseStripe)

```

Comment:
Note that both stripe invoice, stripe checkout, vipps and poweroffice might be used at the same time..

Reply:
Yes, I'm aware. Just leaving it this way for the time-being.

Reply:
Now I got what you meant. The `else` was unintentional 😅